micros/modules/system/boot/runit/services.nix

```
@@ -38,16 +38,29 @@
       runScript = mkOption {
         type = types.str;
         default = "#!${pkgs.runtimeShell}";
+        description = ''
+          Script ran on service startup. Creates the /etc/service/<name>/run file.
```

```suggestion
          Script ran on service startup. Creates the {file}`/etc/service/<name>/run` file.
```

Let's use nixos-render-doc syntax features in option descriptions.